containers/About/index.tsx

```diff
@@ -16,44 +18,65 @@ const About = () => {
     <div id="about" className="w-full py-24 flex flex-col justify-center ">
       <SectionHeading text={"About Me"} index={1} />
       <div className="flex flex-col-reverse sm:grid sm:grid-cols-5 sm:gap-4">
-        <div className="sm:col-start-1 sm:col-end-4 leading-normal text-lg text-blueGray-500 dark:text-gray-600">
-          <p className="mb-2">
-            Hello! My name is Hamza and I enjoy creating things that live on the
-            internet. My interest in web development started back in Lorem ipsum
-            dolor sit amet, consectetur adipiscing elit. Mauris arcu ipsum,
-            rhoncus sed gravida et, luctus eu enim. Donec a nunc placerat velit
-            vestibulum porttitor sit
+        <div className="sm:col-start-1 sm:col-end-4 leading-normal text-xl text-blueGray-500 dark:text-gray-600">
+          <p className="mb-4">
+            I'm currently a student at{" "}
```

Comment:
all of this along with the jumbotron info should be dynamic. Can leave here for now but will need to shift later

Reply:
I'll create an issue.